lib/perl/centreon/script/centcore.pm

```diff
@@ -329,7 +352,7 @@ sub checkSSHPort($) {
     if (defined($value) && $value) {
         $port = $value;
     } else {
-        $port = 22;
+        $port = 22;gt
```

Comment:
what does "gt" that mean ?

Reply:
Probably some typo that passed by, I will fix it, thanks!